executor/union_scan.go

```diff
@@ -147,10 +152,7 @@ func (us *UnionScanExec) getSnapshotRow() (*Row, error) {
 			if us.snapshotRow == nil {
 				break
 			}
-			if len(us.snapshotRow.RowKeys) != 1 {
-				return nil, ErrRowKeyCount
-			}
-			snapshotHandle := us.snapshotRow.RowKeys[0].Handle
+			snapshotHandle := us.snapshotRow.Data[len(us.snapshotRow.Data)-1].GetInt64()
```

Comment:
Why snapshotRow's handle is always in the tail ? I think it should be `us.handle` ?